taskt.Commands/List Commands/AddListItemCommand.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.Office.Interop.Outlook;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Attributes.ClassAttributes;
+using taskt.Core.Attributes.PropertyAttributes;
+using taskt.Core.Command;
+using taskt.Core.Enums;
+using taskt.Core.Infrastructure;
+using taskt.Core.Script;
+using taskt.Core.Utilities.CommonUtilities;
+using taskt.Engine;
+using taskt.UI.CustomControls;
+using Exception = System.Exception;
+
+namespace taskt.Commands
+{
+    [Serializable]
+    [Group("List Commands")]
+    [Description("This command adds an item to an existing List variable.")]
+    public class AddListItemCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [PropertyDescription("List")]
+        [InputSpecification("Provide a List variable.")]
+        [SampleUsage("{vList}")]
+        [Remarks("Any type of variable other than list will cause error.")]
```

Comment:
Should change to "List" to match the other corrections.

Reply:
done

Reply:
done